### DISCUSSION TITLE
Aliens artwork

### DISCUSSION THREAD

**Discussion by GPaso** [endless-sky/endless-sky#9416] [General]
Is there already an alien artwork on how they look?

I found just this coloured height comparsion:
https://steamcommunity.com/sharedfiles/filedetails/?id=2877403856

**Comment by Saugia** [comment]
We don't have a specific or "canonized" artwork for the aliens in the game so that the players can imagine their own headcanons on what they might look like. The most we do is describe how they look in writing, which could be taken many different ways. There are some who have attempted to render their own idea in artwork, but there is no true image or answer for the sake of imagination.

**Reply by GPaso** [reply]
Would it be interesting to make those artworks for reference and put them in the fandom wiky? 
I am no designer but maybe i could use an AI and put the generated artworks to vote here.

**Reply by mOctave** [reply]
See https://github.com/endless-sky/endless-sky/discussions/8787#discussioncomment-6042415. You are, of course, free to make whatever images you want, but if they're AI-generated they won't be used in the vanilla game and are unlikely to be used on the wiki, either.

Also, as Saugia said, not having images for aliens or most characters is intentional: that way the player decides what people look like and aren't shocked later when their headcanon opinion doesn't match with the character's "official" look.

**Comment by GPaso** [comment]
Understood! thanks for the clarification!
